.gitignore

```diff
@@ -1,5 +1,5 @@
-/build
-/target
+/build*/
```

Comment:
I forgot we had a `buildtools` directory, so this probably isn't a good idea.
We'd have to do something else, maybe:
```.gitignore
/build/
/build_*/
```
Or just leave this alone and let developers configure `$GIT_DIR/info/exclude` for their own needs

Reply:
Oh, you're right. Changed to `/build/` with commit 19664728.